app/models/purchase.rb

```diff
@@ -105,7 +100,7 @@ def csv_export_attributes
   private
 
   def combine_duplicates
-    Rails.logger.info "Combining!"
+    Rails.logger.info "[!] Purchase.combine_duplicates: Combining!"
```

Comment:
I guess if we prefix logger is [!] that makes it easy to grep

Reply:
see above, re: your idea about custom logger. I think that's the way to go

Reply:
Isn't this what the log priority is supposed to handle for us? So if it's worthy of a `[!]`, should it be `warn` instead of `info`? Using `[!]` here but not for an error doesn't feel right.

Reply:
That's a good point. I forgot you can set different logging levels when you output it!

I still think that having a separate log that tracks user activity might be a good idea so that we don't have to parse massive log files with request stuff just to find some user transactions.

Reply:
#1002